apps/system/js/advanced_telemetry.js

```diff
@@ -570,23 +570,21 @@
   // This method loops through the payload of histograms and converts each
   // histogram payload to the 'dict' format required by the server for parsing.
   AdvancedTelemetryPing.prototype.packHistograms = function() {
-    var nameAddonHist = this.packet.payload.addonHistograms;
-    var nameKeyHist = this.packet.payload.keyedHistograms;
-
-    // Pack the Addon Histograms.
-    for (var addon in nameAddonHist) {
-      for (var hist in nameAddonHist[addon]) {
-        nameAddonHist[addon][hist] =
-          this.packHistogram(nameAddonHist[addon][hist]);
-      }
-    }
+    function packHistogramType(level, self) {
```

Comment:
How did you come up with 'level'? It represents a collection of histograms, correct? Is 'histograms' too long?